runtime/scripts/scorm-storage.js

```diff
@@ -20,12 +20,14 @@ var scorm = Numbas.storage.scorm = {};
  */
 var SCORMStorage = scorm.SCORMStorage = function()
 {
+    Numbas.storage.lmsConnected = true;
```

Comment:
I would prefer to have this as `false` to begin with, and set it to `true` if the SCORM API is found.

Reply:
In fact, this could be set to false in the prototype of `SCORMStorage`, which would let you add a documentation tag.